lib/mix/lib/mix/tasks/compile.elixir.ex

```diff
@@ -81,4 +82,14 @@ defmodule Mix.Tasks.Compile.Elixir do
     dest = Mix.Project.compile_path()
     Mix.Compilers.Elixir.clean(manifest(), dest)
   end
+
+  defp xref_exclude_opts(opts, project) do
+    exclude = List.wrap(project[:xref][:exclude])
```

Comment:
Is `:xref, :exclude` going to be deprecated in future version of Elixir in favor of `:no_warn_undefined`? (to add a TODO)